setup.py

```diff
@@ -62,9 +62,10 @@ def get_setup_version(reponame):
 
 
 # TODO: need to verify this list (not sure all needed), plus some
-# versions may be needed
+# versions may be needed.
 extras_require={
     'recommended': [
+        'paramnb',
```

Comment:
What's paramnb needed for?

Reply:
Just looked, ``notebooks/Dashboard_Demo.ipynb`` should just be deleted.

Reply:
Ok, have done that. 

What about Colocated_cube_pair.ipynb and CubeExplorer_Prototype.ipynb? Can they go onto the website, or they're not ready/suitable for website? (Either is fine, just checking.)

Reply:
At least on travis/linux, on this branch (where I've been changing the dependencies...), I get this:

Colocated_cube_pair:
<img width="1118" alt="screen shot 2018-04-30 at 11 54 16 am" src="https://user-images.githubusercontent.com/1929/39424217-6f6f21fc-4c6d-11e8-8ad2-ddf44eca3d70.png">

CubeExplorer_Prototype:
<img width="1080" alt="screen shot 2018-04-30 at 11 53 50 am" src="https://user-images.githubusercontent.com/1929/39424236-88970e60-4c6d-11e8-9897-3994c8c37529.png">

Haven't tried running them locally.